lib/linguist/vendor.yml

```diff
@@ -127,44 +127,44 @@
 - (^|/)jquery\.(ui|effects)\.([^.]*)\.(js|css)$
 
 # jQuery Gantt
-- jquery.fn.gantt.js
+- jquery\.fn\.gantt\.js
 
 # jQuery fancyBox
-- jquery.fancybox.(js|css)
+- jquery\.fancybox\.(js|css)
 
 # Fuel UX
-- fuelux.js
+- fuelux\.js
 
 # jQuery File Upload
 - (^|/)jquery\.fileupload(-\w+)?\.js$
 
 # jQuery dataTables
-- jquery.dataTables.js
+- jquery\.dataTables\.js
 
 # bootboxjs
-- bootbox.js
+- bootbox\.js
 
 # pdf-worker
-- pdf.worker.js
+- pdf\.worker\.js
 
 # Slick
 - (^|/)slick\.\w+.js$
 
 # Leaflet plugins
 - (^|/)Leaflet\.Coordinates-\d+\.\d+\.\d+\.src\.js$
-- leaflet.draw-src.js
-- leaflet.draw.css
-- Control.FullScreen.css
-- Control.FullScreen.js
-- leaflet.spin.js
-- wicket-leaflet.js
+- leaflet\.draw-src\.js
+- leaflet\.draw\.css
+- Control\.FullScreen\.css
+- Control\.FullScreen\.js
+- leaflet\.spin\.js
+- wicket-leaflet\.js
 
 # Sublime Text workspace files
-- .sublime-project
-- .sublime-workspace
+- \.sublime-project
+- \.sublime-workspace
 
 # VS Code workspace files
-- .vscode
+- ^\.vscode/
```

Comment:
Should this also match `/somedir/.vscode`?

Reply:
Good point, yes, I suspect it should.

Reply:
Could you update the regex so that it does?